template/README.md

```diff
@@ -96,6 +96,15 @@ $ npm run serve
 
 Open [localhost:8000](http://localhost:8000/)
 
+### Writing SCSS
+Auro has implemented [CSScomb](https://www.npmjs.com/package/csscomb) for style formatting enforcement in addition to the [linting requirements](https://auro.alaskaair.com/webcorestylesheets/linter).
```

Comment:
```suggestion
Auro now supports [CSScomb](https://www.npmjs.com/package/csscomb) for style formatting enforcement in addition to the [linting requirements](https://auro.alaskaair.com/webcorestylesheets/linter).
```

Reply:
This wording informs the user that this just happened. Which is correct today but won't be in a few months. I chose wording that reads correctly no matter how far in the future someone looks at the document as we certainly won't be editing this language down the line.